vita3k/app/src/app_init.cpp

```diff
@@ -229,6 +279,53 @@ void init_paths(Root &root_paths) {
     fs::create_directories(root_paths.get_patch_path());
 }
 
+#ifdef __LINUX__
+static float fetch_x11_display_dpi() {
```

Comment:
:thinking: SDL3 does more than just retrieve a value from "Xft.dpi", you can see there are other potential values to check: https://github.com/libsdl-org/SDL/blob/c6b5c21aaf66dc8fb4214e7bebb1258084abfae6/src/video/x11/SDL_x11modes.c#L49-L119
Do you think if it will be worth it to do the same logic?

Reply:
Thank you. Since there's no guarantee that we can obtain the Xft.dpi value from the system, having more fallback options is better. The fallbacks included in SDL3's implementation should be quite reliable. I will add fallbacks with reference to the cited code.

Reply:
After review, getting values from XSETTINGS seems too costly compared to its benefits. We would need to either add a dependency on libxsettings-client or implement direct value access, but the benefits don't seem worth adding these for this purpose.
While GDK_SCALE is also not widely used currently, it's not a bad option since it has minimal dependencies. Therefore, I only added GDK_SCALE as a fallback.